x-pack/plugins/apm/server/routes/alerts/register_apm_rule_types.ts

```diff
@@ -50,6 +52,14 @@ export const apmRuleTypeAlertFieldMap = {
     type: 'keyword',
     required: false,
   },
+  [ERROR_GROUP_ID]: {
+    type: 'keyword',
+    required: false,
+  },
+  [ERROR_GROUP_NAME]: {
+    type: 'text',
+    required: false,
+  },
```

Comment:
@sqren I would like to get your opinion on adding these fields in the index mapping and whether their types are accurate.